letsencrypt/cli.py

```diff
@@ -744,6 +750,8 @@ def create_parser(plugins, args):
                 help="Obtain and install certs using Apache")
     helpful.add(None, "--nginx", action="store_true",
                 help="Obtain and install certs using Nginx")
+    helpful.add(None, "--standalone", action="store_true",
+                help='Obtain certs using a "standalone" webserver on port 443.')
```

Comment:
Nit: Standalone now supports both SimpleHTTP and DVSNI. Perhaps just change this to `Obtain certs using a "standalone" webserver.`?